contributor_docs/contributor_guidelines.md

```diff
@@ -6,14 +6,17 @@ If you are looking to contribute outside of the p5.js repositories (writing tuto
 This is a fairly long and comprehensive document but we will try to deliniate all steps and points as clearly as possible. Do utilize the table of contents, the browser search functionality (`Ctrl + f` or `Cmd + f`) to find sections relevant to you. Feel free to skip sections if they are not relevant to your planned contributions as well.
 
 # Table of Contents
+- [Contributor Guidelines](#contributor-guidelines)
+- [Table of Contents](#table-of-contents)
```

Comment:
I'm not sure these two are necessary as they don't really need to be navigated to from the table of contents

Reply:
Updated!